style.css

```diff
@@ -1,15 +1,27 @@
+body {
+  background: #e5e5e5;
+}
+
```

Comment:
- [ ] [OPTIONAL] Let's add `box-sizing: border-box` property to every element using `*`. This property makes it easier to control elements' width. You can read more about it [here](https://www.freecodecamp.org/news/how-to-use-the-css-box-sizing-property/).

Reply:
could you please help me with the syntax?